lib/bitters/generator.rb

```diff
@@ -10,13 +10,14 @@ class Generator < Thor
     map ['delete'] => :remove
 
     desc 'install', 'Install Bitters into your project'
-    method_options path: :string
+    method_options path: :string, syntax: :string
     def install
       if bitters_files_already_exist?
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/base."
+        puts "Converted files to Sass syntax." if options[:syntax].downcase == 'sass'
```

Comment:
The thoughtbot styleguide also suggests not using conditional modifiers (the conditional at the end of the line), so this might be better as:
```
if options[:syntax].downcase == "sass"
  puts "Converted files to Sass syntax."
end
```